Documentation/ceph-object-store-crd.md

```diff
@@ -119,3 +132,32 @@ execution attack.
 Rook will not overwrite an existing `mime.types` ConfigMap so that user modifications will not be
 destroyed. If the object store is destroyed and recreated, the ConfigMap will also be destroyed and
 created anew.
+
+## Health settings
+
+Rook-Ceph will be default monitor the state of the object store endpoints.
+The following CRD settings are available:
+
+* `healthCheck`: main object store health monitoring section
+
+Here is a complete example:
+
+```yaml
+healthCheck:
+  bucket:
+    enabled: true
+    interval: 60s
+```
+
+The endpoint health check procedure is the following:
+
+1. purge the s3 object by default
+2. create an S3 user
```

Comment:
nit: For documentation I think it's more common to capitalize bullet points, but maybe that's just me